### PR TITLE
docs: add casdoor

### PR DESCRIPTION
**Pull request by jakiuncle**
Casdoor is a Support OIDC, OAuth 2.0, SAML, LDAP
With [Casbin](https://casbin.org/) based authorization management, Casdoor supports ACL, RBAC, ABAC, RESTful accessing control models
Front-end and back-end separate architecture, Casdoor supports high concurrency, provides web-based managing UI and have mature solution for springboot project https://casdoor.org/docs/integration/spring-boot
Casdoor supports Github, Google, QQ, WeChat third-party applications login, and support the extension of third-party login with plugins.
Phone verification code, email verification code and forget password features.
Accessing logs auditing and recording.
Casdoor supports integration with existing systems using db sync method, users can transition to Casdoor smoothly.
Casdoor supports mainstream databases: MySQL, PostgreSQL, SQL Server etc, and support the extension of new database with plugins.

#https://github.com/bytebase/bytebase/pull/5868

### CODE REVIEW

**Comment by d-bytebase** [comment]
> We require contributors to sign our Contributor License Agreement, and we don't have yours on file. In order for us to review and merge your code, please please sign [CLA](https://forms.gle/zcNM9TL1iVU1fQ1G9) and add your name to [contributors list](https://github.com/bytebase/clabot-config/blob/main/.clabot).

Welcome! Would you mind adding yourself to https://github.com/bytebase/clabot-config/blob/main/.clabot ? Thanks!

File: content/docs/administration/sso/oauth2.md [hunk 1]
```diff
@@ -88,3 +88,14 @@ Bytebase provides templates for configuring built-in OAuth providers.
 3. Update the **Client ID** and **Client secret** fields with the OAuth 2.0 client credential you just created.
 4. After filling in all the required fields, try to click **Test connection**.
 5. If everything is OK, click the **Create** button.
+
+### Casdoor
```
Comment (d-bytebase) [inline content/docs/administration/sso/oauth2.md:92]: This doc is used for mapping from console. It's the best way to introduce Casdoor sso integration by adding a blog at https://www.bytebase.com/blog

Comment (hsluoyz) [reply content/docs/administration/sso/oauth2.md:92]: @d-bytebase is it OK to add an item besides: https://www.bytebase.com/docs/administration/sso/oidc#keycloak ? There are already a lot of similar vendors there.

Comment (d-bytebase) [reply content/docs/administration/sso/oauth2.md:92]: Aah, yes. Thanks for pointing it out. Let's append it.

Comment (hsluoyz) [reply content/docs/administration/sso/oauth2.md:92]: great, thanks!

**Pull request closed without merging**